src/core_modules/capture-core/components/Pages/Enrollment/EnrollmentPageDefault/EnrollmentPageDefault.component.js

```diff
@@ -40,7 +41,7 @@ export const EnrollmentPageDefaultPlain = ({ program, classes }: PlainProps) =>
                 />
             </div>
             <div className={classes.rightColumn}>
-                [placeholder profile widget]
+                <ProfileWidget attributes={attributes} />
```

Comment:
Hey, 
We discussed that we want to make the profile widget component self-contained to be able to use it in other apps. Does this mean it must pull its own content and not receive it from the pattent? 
Thanks!

Reply:
that makes sense! I will change that